module/Database/view/database/member/email-welcome.phtml

```diff
@@ -0,0 +1,96 @@
+</p>
+
+<p>
+    <b>Members</b><br/>
+    In this section, you can find plenty of different important shortcuts to other parts of the website. For example; you can look at an overview of your profile, look up another member if you can’t remember his name, see when the next General Members Meeting (GMM) is, access your SudoSOS account with which you can buy drinks and food (you have to agree to the terms of use before you can buy stuff), and many other shortcuts.
+    Be sure to check it out!
+    And of course, on our homepage, you can find numerous of these sections in short. A few of the upcoming events, the birthdays of today, the photo of the week, a poll, and of course random infima. So as you can see, there’s a lot to see and do at our website, so be sure to check it out every once in a while!
+</p>
+
+<p>
+    <b>First Year Committees</b><br/>
+    GEWIS has many committees and fraternities, we even set-up committees dedicated to first year students every year. As a first year committee you will be experiencing first hand what being an active GEWIS member is like! Together with your fellow first year students you will be fleshing out your own identity by creating your very own logo and custom clothing.
+    You will also get the opportunity to organize some very fun activities for GEWIS and you can even be a part of workgroups with other first year committee members with whom you will be organizing parties, parent days or even a cantus!
+    Furthermore, each semester a scoring competition is held to see which first year committee is the very best of them all.
+    Collect points with your committee by passing courses, attending activities and much more!
+</p>
+
+<p>
+    <b>Joining a first year committee</b></br/>
+    In the first quartile, an information meeting will be held.  Here, first year committees from previous years will tell you about their fun experiences.
+    A second meeting, the formation meeting, will be held to form the first year committees. Not to worry though, it's also possible to join an FYC later in the year.
+    We hope to see you at these meetings!
+</p>
+
+<p>
+    <b>Ordering books</b><br/>
+    By becoming a GEWIS member you have the right to specialized discount! By navigating to <a href="https://gewis.nl/books">gewis.nl/books</a> you can find your study and see which books will be used for your courses.
+</p>
+
+<p>
+    <b>GEWIS on Social Media</b><br/>
+    Next to our own photo section on the website, you can also find us on social media. We have Instagram, Facebook & LinkedIn. Additionally, we have a weekly newsletter, for which you are automatically subscribed. You will get your first newsletter soon! 
+</p>
+
+<p>
+    With kind regards,<br/>
+    Daan Verkade<br/>
```

Comment:
I do not think we want to keep updating this every year just to change the secretary.

Reply:
Fixed